addons/account/static/src/components/bills_upload/bills_upload.xml

```diff
@@ -50,28 +50,26 @@
 
     <t t-name="account.ListView.Buttons" t-inherit="web.ListView.Buttons" t-inherit-mode="primary" owl="1">
         <xpath expr="//*[@class='btn btn-primary o_list_button_add']" position="after">
-            <AccountFileUploader extraContext="props.context">
-                <t t-set-slot="toggleButton">
-                    <button type="button" class="btn btn-secondary o_button_upload_bill">
-                        Upload
-                    </button>
-                </t>
-            </AccountFileUploader>
+            <t t-call="account.AccountViewUploadButton"/>
         </xpath>
     </t>
 
     <t t-name="account.KanbanView.Buttons" t-inherit="web.KanbanView.Buttons" t-inherit-mode="primary" owl="1">
         <xpath expr="//*[@class='btn btn-primary o-kanban-button-new']" position="after">
-            <AccountFileUploader extraContext="props.context">
-                <t t-set-slot="toggleButton">
-                    <button type="button" class="btn btn-secondary o_button_upload_bill">
-                        Upload
-                    </button>
-                </t>
-            </AccountFileUploader>
+            <t t-call="account.AccountViewUploadButton"/>
         </xpath>
     </t>
 
+    <t t-name="account.AccountViewUploadButton" owl="1">
+        <AccountFileUploader extraContext="props.context">
+            <t t-set-slot="toggleButton">
+                <button type="button" class="btn btn-secondary o_button_upload_bill">
+                    Upload
+                </button>
+            </t>
+        </AccountFileUploader>
+    </t>
```

Comment:
Are the changes in this file needed for the fix or is it just a refactor?

Reply:
No, not required for the fix, just a small cleanup of duplicated code

Reply:
I have changed some code - and since it was duplicated - I have extracted it to a template